### PR TITLE
Fixed For Endpoint Enrichment Playbooks

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9043

## Description
As part of the common playbook update effort, I reviewed the 'Endpoint Enrichment - Generic v2.1_6_8' playbook and noticed the following issues:
1- Playbook input configurations - updated the 'EndpointID' and 'IPAddress' playbook input values.
2- Playbook input descriptions - the 'UseReputationCommand' playbook input description now includes 'The default value is true'.
3- Automation inputs configurations - added the 'unique' transformer to the script arguments within tasks number 37, 9, 32, 38, and 34.
4- Unclear task descriptions - The descriptions of tasks 33 and 43 have been updated.
5- Missing playbook outputs.

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml:1]: 1. If you are adding more integrations such as Cortex XDR, maybe we should add also mgmt solutions such as SCCM and Jamf
2. Most of these integrations (CS, Cortex XDR, etc) have also the default `!endpoint` command - which brings duplicated outputs when running both the reputation and the vendors commands. Wondering if we should do so... WDYT?
3.

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_33.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+
+##### Endpoint Enrichment - Generic v2.1
+
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_33.md:5]: please mention that you added also XDR here

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml [hunk 2]
```diff
@@ -1327,51 +1497,492 @@ outputs:
...
-- Generic Polling Test
-- Send Investigation Summary Reports - Test
-- Detonate File - No Files test
-- Test Convert file hash to corresponding hashes
-- Detonate File - Generic Test
 - Endpoint Enrichment - Generic v2.1 - Test
-- Isolate and unisolate endpoint - test
-- Block IP - Generic V3_Test
 fromversion: 5.0.0
 toversion: 6.7.9
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml:1988]: Should even touch this PB if this is under 6.8?

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml [hunk 3]
```diff
@@ -861,87 +807,205 @@ tasks:
...
+      - - ignorecase: true
+          left:
+            iscontext: true
+            value:
+              complex:
+                root: inputs.UseReputationCommand
+          operator: isEqualString
+          right:
+            value:
+              simple: "True"
```
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml:893]: should we add here ignore case?

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml:1]: I didn't add the XDR core, Ido added it a month ago.
Regarding the duplications the !endpoint command creates - I think that is why the 'UseReputationCommand' default value is 'False'...

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml (hunk 2 above)
Comment (TalNos) [reply Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml:1988]: I got this PB and I also need to update the 'Endpoint_Enrichment_-_Generic_v2.1_6_8.yml' playbook

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**